autoscale/src/main/resources/schema/app/20221110093257_CB-19545_Add_new_entity_to_record_autoscaling_triggers.sql

```diff
@@ -0,0 +1,24 @@
+-- // CB-19545 Add new entity to record autoscaling triggers
+-- Migration SQL that makes the change goes here.
+create sequence if not exists scaling_trigger_id_seq;
+
+create table if not exists scaling_trigger (
+	id bigint default nextval('scaling_trigger_id_seq'::regclass) not null
+	constraint scaling_trigger_pkey primary key,
+	trigger_crn varchar(255),
+	flow_id varchar(255),
+	start_time timestamp,
+	yarn_metrics text,
+	trigger_status varchar(255),
+	cluster_id bigint,
+
+	constraint fk_scaling_trigger_cluster_id foreign key (cluster_id) references cluster(id)
+);
+
+create unique index if not exists idx_trigger_crn on scaling_trigger(trigger_crn);
```

Comment:
Would we need some kind of an index on the startTime as well - given it is used extensively for various queries.